<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetMaker</class>
 <widget class="QMainWindow" name="PresetMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <italic>false</italic>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>PresetMaker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QGroupBox" name="InputTypesBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>111</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Input Types</string>
    </property>
    <widget class="QPushButton" name="BaseColorButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Base Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NormalsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Normals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RoughnessButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Roughness</string>
     </property>
    </widget>
    <widget class="QPushButton" name="MetalnessButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Metalness</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AOButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HeightButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QPushButton" name="EmissiveButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Emissive</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Output Types</string>
    </property>
    <widget class="QPushButton" name="RGBButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RGB (3 Ch)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RGBAButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RGBA (4 Ch)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Preset Name:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SavePresetButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>SAVE</string>
    </property>
   </widget>
   <widget class="QComboBox" name="NameBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Medium</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Create New...</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>151</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Outputs</string>
    </property>
    <widget class="QGroupBox" name="OutputNamesBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>151</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ChannelGroup">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>331</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Inputs + Channels</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetInputsButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET INPUTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetOutputsButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET OUTPUTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RenameButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Light</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>RENAME</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
